packages/commons/src/index.ts

```diff
@@ -1,4 +1,5 @@
 export * from './utils/lambda';
 export * from './Utility';
+export * from './tests/e2e';
```

Comment:
If we export it here, does it mean the module gets included in the published package? If that's the case maybe we don't want that

Reply:
Good point. I just saw Florian's PR that he import directly (e.g. `../../../commons/tests/utils/cdk-cli`) . This makes more sense as the code is internal.